js/mexc.js

```diff
@@ -2459,4 +2459,95 @@ module.exports = class mexc extends Exchange {
         const sorted = this.sortBy (rates, 'timestamp');
         return this.filterBySymbolSinceLimit (sorted, symbol, since, limit);
     }
+
+    async fetchLeverageTiers (symbol = undefined, params = {}) {
+        await this.loadMarkets ();
+        const symbolDefined = (symbol !== undefined);
+        let market = undefined;
+        if (symbolDefined) {
+            market = this.market (symbol);
+            if (!market['contract']) {
+                throw new BadRequest (this.id + ' fetchLeverageTiers symbol supports contract markets only');
+            }
+        }
+        const result = {};
+        const response = await this.contractPublicGetDetail ();
```

Comment:
We have to propagate the `params` as the only argument to the implicit call here.